src/rp2_common/pico_standard_link/crt0.S

```diff
@@ -74,6 +74,11 @@ __vectors:
 .word isr_irq30
 .word isr_irq31
 
+// all default exception handlers do nothing, and we can check for them being set to our
+// default values by them pointing to between __defaults_isrs_start and __default_isrs_end
```

Comment:
I initially thought "by them pointing to" was a typo for "by pointing them to", but then I reread it a couple of times and figured out what it actually meant :wink: 
Perhaps "by seeing if they point to somewhere" would be clearer?